docs/guide/content/user_guide/finding_data.md

```diff
@@ -10,9 +10,9 @@ The application supports two ways of finding datasets from remote repositories:
 
 !!! note "Supported Repositories"
 
-    OnDemand Loop currently supports two repository connectors: **Dataverse** and **Zenodo**.  
+    OnDemand Loop currently supports two repository connectors: **Dataverse** and **Zenodo** (contributions are welcome to add more).  
     Only **public datasets** are supported for download at this time.  
-    Support for **draft datasets**, **private access**, and **additional repositories** is planned for future releases.
+    Support for **[draft datasets](https://github.com/IQSS/ondemand-loop/issues/310)**, and **private access** is planned for future releases.
```

Comment:
I don't think we want to imply that we are going to code up integrations with Figshare, etc.

Also, when talking about plans, it would be nice to link to a roadmap.

I couldn't find an issue about private access to link to. 🤷 Is it planned? Is it on a roadmap?

Reply:
The GitHub issues with label `dev.Release1` are those considered to be implemented first, so we could say that is our current scope, but we may remove some of the items and implement others proposed in the reviews. 

[Features - Add repository management](https://github.com/IQSS/ondemand-loop/issues/299) will help accessing private Datasets and actually the one you mentioned [draft datasets](https://github.com/IQSS/ondemand-loop/issues/310) will use the API key to access private data from Draft datasets. We will update the guide once the latter PR is merged